src/app/components/speakers/speakers.js

```diff
@@ -1,47 +1,43 @@
-
 import React from 'react';
 import { speakersData } from "assets/data/speakers-info.js";
 import './speakers.scss'
 
-const Speaker = (name, title, photo, index) => {
-    return (
-        <div className="speaker-img-wrapper">
-            <img 
-                src = {require('assets/images/speaker-photos/'+photo+'.jpeg')}
-                alt = {name + "'s profile picture"}
-            />
-            <div className="text">
-                <h4>{name}</h4>
-                <p>{title}</p>
-            </div>
-        </div>
-    );
-}
+export default function Speakers() {
 
-const renderSpeakers = (speakers) => {
-    return (
-        <div className="speaker-photos">
-            {speakers.map((speaker, index) => 
-                Speaker(speaker.name, speaker.title, speaker.photo, index)
-            )}
-        </div>
-    ) 
-}
+    function Speaker(name, title, photo, index) {
+        return (
+            <div className="speaker-img-wrapper">
+                <img 
+                    src = {require('assets/images/speaker-photos/'+photo+'.jpeg')}
+                    alt = {name + "'s profile picture"}
+                />
+                <div className="text">
+                    <h4>{name}</h4>
+                    <p>{title}</p>
+                </div>
+            </div>
+        )
+    }
 
-export default class Speakers extends React.Component {
-    render() {
+    function renderSpeakers(speakers) {
```

Comment:
this is also good 😎 just needs the `const renderSpeakers = (speakers) => {...}` and skipping the return from one of the other comments